pkg/monitor/agent.go

```diff
@@ -55,3 +64,77 @@ func resolveAgentType(t AgentNotification) string {
 func (n *AgentNotify) DumpInfo() {
 	fmt.Printf(">> %s: %s\n", resolveAgentType(n.Type), n.Text)
 }
+
+// PolicyUpdateNotification structures update notification
+type PolicyUpdateNotification struct {
+	Labels    []string `json:"labels,omitempty"`
+	Revision  uint64   `json:"revision,omitempty"`
+	RuleCount int      `json:"rule_count"`
+}
+
+// PolicyUpdateRepr returns string representation of monitor notification
+func PolicyUpdateRepr(rules api.Rules, revision uint64) (string, error) {
+	labels := make([]string, 0, len(rules))
+	for _, r := range rules {
+		labels = append(labels, r.Labels.GetModel()...)
```

Comment:
Only append if `r.Labels` is not empty. It's not useful to just return a bunch of `nil` values.

Reply:
if `r.Labels.GetModel()` returns empty slice no elements will be appended to `labels`.